src/resource/archive/ArchiveManager.cpp

```diff
@@ -5,7 +5,9 @@
 
 #include "resource/archive/Archive.h"
 #include "resource/archive/OtrArchive.h"
+#ifndef PLATFORM_IOS
 #include "resource/archive/O2rArchive.h"
+#endif
```

Comment:
still curious about the ios not supporting libzip thing

Reply:
I tried for hours building libzip but i didnt manage to do it